realm/realm-library/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -411,6 +411,12 @@ public TableQuery isEmpty(long[] columnIndices) {
         return this;
     }
 
+    public TableQuery isNonEmpty(long[] columnIndices) {
+        nativeIsNonEmpty(nativePtr, columnIndices);
```

Comment:
@cmelchior I totally agree with you that this is redundant. I thought it would be good to make the particular function on par with others and, mostly, I want to walk JNI path to see how things are working down there.